config.ru

```diff
@@ -1,4 +1,6 @@
 # This file is used by Rack-based servers to start the application.
 
-require ::File.expand_path('../config/environment', __FILE__)
+require_relative "config/environment"
```

Comment:
Under the hood change in 6.1 but we don't need to worry about it much. (There are a bunch of `require_relative`s replacing more elaborate requires.)